docs/en/query_language/functions/string_functions.md

```diff
@@ -152,9 +152,34 @@ Similar to base64Decode, but in case of error an empty string would be returned.
 
 Returns whether to end with the specified suffix. Returns 1 if the string ends with the specified suffix, otherwise it returns 0.
 
-## startsWith(s, prefix) {#function-startswith}
+## startsWith(str, prefix) {#function-startswith}
 
-Returns whether to start with the specified prefix. Returns 1 if the string starts with the specified prefix, otherwise it returns 0.
+Returns 1 whether string starts with the specified prefix, otherwise it returns 0.
+
+```sql
+SELECT startsWith('string', 'str');
```

Comment:
We can replace 'str' to something else to avoid confusion with `startsWith(str, prefix)` above.